gmail/endpoint.bal

```diff
@@ -80,16 +84,19 @@ public client class Client {
     # Create the raw base 64 encoded string of the whole message and send it as an email from the user's
     # mailbox to its recipient.
     #
-    # + userId - The user's email address. The special value **me** can be used to indicate the authenticated user.
     # + message - MessageRequest to send
     # + threadId - Optional. Required if message is expected to be send The ID of the thread the message belongs to.
     # (The Subject headers must match)
+    # + userId - The user's email address. The special value **me** can be used to indicate the authenticated user.
     # + return - If successful, returns Message record of the successfully sent message. Else return error.
     @display {label: "Send Message"} 
-    remote function sendMessage(@display {label: "Email Address"} string userId,
-                                @display {label: "Message Request to Send"} MessageRequest message,
-                                @display {label: "Thread Id"} string? threadId = ())
+    remote function sendMessage(@display {label: "Message Request to Send"} MessageRequest message,
```

Comment:
Shall we update the label to `Send Message` ?

Reply:
Ack